src/app/pages/swap/components/swap-toggle-button.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box } from '@stacks/ui';
+import { useFormikContext } from 'formik';
+
+import { SwapIcon } from '@app/components/icons/swap-icon';
+
+import { SwapFormValues } from '../hooks/use-swap';
+
+interface SwapToggleButtonProps {
+  onSetIsSendingMax(value: boolean): void;
+}
+export function SwapToggleButton({ onSetIsSendingMax }: SwapToggleButtonProps) {
+  const { setFieldValue, values } = useFormikContext<SwapFormValues>();
+
+  async function onToggleSwapAssets() {
+    onSetIsSendingMax(false);
+    // const prevAmountFrom = values.swapAmountFrom;
```

Comment:
Some stuff to delete here

Reply:
Yeah, was leaving these until I get an answer on the `TODO` below. It might be that the amounts should swap too?